share/man/kodev.1

```diff
@@ -0,0 +1,130 @@
+.TH KODEV 1
+.SH NAME
+kodev \- Kore project management tool
+
+.SH SYNOPSIS
+.BR kodev
+[\fIOPTION\fR] ...
+
+.SH DESCRIPTION
+This documentation describes the application management tools for building and
+running a kore project. For information regarding the technical interface and
+C bindings, look to
+.BR KORE(3)
+where these functions and structures are described.
+.BR kore
+projects may be managed using the following OPTIONS;
+
+.BR create
+.RS
+Create a new application skeleton with the name that is passed to it. This will
+create a new directory with all the files required to begin hacking about.
+This will also generate a self-signed certificate \fIunless\fR \fBkore\fR was
+built with \fBNOTLS=1\fR. Note this certificate should not be used for
+production.
+.RE
+
+.BR build
+.RS
+Build the application. The properties of this build are read from
+.BR conf/build.conf
+and the semantics of which are discussed in \fBCONFIGURATION FILES\fR
+.RE
+
+.BR run
+.RS
+Run an application (\-fnr implied). The current directory is the path of the
+project to be run. This will read from the \fBconf/app.conf\fR discussed in
+\fBCONFIGURATION FILES\fR unless otherwise specified. This will build the
+application\-\-if necessary\-\-and start it in the foregound. \fB^C\fR
+will stop the process.
+.RE
+
+.BR reload
+.RS
+Reload the application (SIGHUP sent). Reads the application run configuration
+which would be read in a normal run on a reload.
```

Comment:
So SIGHUP actually does not reload the configuration, it tells kore to reload all loaded modules.